src/tests/cli.rs

```diff
@@ -25,7 +25,10 @@ fn bash_error_exit_code() -> Result<(), Box<dyn std::error::Error>> {
 
     // Changes locale to default to prevent locale-specific error messages.
     cmd.env("LC_ALL", "C")
-        .arg("--disable-format")
+        .arg("--disable-postprocessor")
+        .arg("shfmt")
+        .arg("--disable-postprocessor")
+        .arg("bshchk")
```

Comment:
Maybe can be handy `any` as value so all of them are disabled?

Reply:
i'd rather have it support a `*` wildcard

Reply:
it is fine too